### ISSUE TITLE
new term request: glutathione specific gamma-glutamylcyclotransferase activity

### ISSUE THREAD

**Issue by kaxelsen** [geneontology/go-ontology#14075]
Hi,
I would like to have this new term created. It is similar to 
GO:0003839  gamma-glutamylcyclotransferase activity
but the definition is different:
Catalysis of the reaction: glutathione = 5-oxoproline + L-cysteinylglycine.
The activity has not been assigned an EC number yet.

It is relevant for CHAC1 (e.g. Q9BUX1) and CHAC2 (Q8WUX2) as well as CHAC_YEAST (P32656) and has been characterized in: pmid: 27913623 and pmid:23070364.

Best wishes,
Kristian

**Comment by ukemi** [comment]
Hi Kristian,

Have you requested an EC number for this?

**Comment by kaxelsen** [comment]
Yes, I have created an EC entry that has the internal working number EC 4.3.2.b. It will most likely take 3-4 months before it is assigned its final EC number.

It would also be nice to have a GO term for 
Gamma-glutamylaminecyclotransferase
I have also created an EC entry (EC 4.3.2.c) for this activity. 
Definition: epsilon-(L-gamma-glutamyl)-L-lysine = L-lysine + 5-oxo-L-proline.
Relevant for GGACT (e.g. Q9BVM4)
Refs: pmid:6107907 and  pmid:20110353


The last thing I have done is to transfer EC 2.3.2.4 (linked to GO:0003839 gamma-glutamylcyclotransferase activity) to EC 4.3.2.d, since the current EC number is a misclassification.
This new classification as a lyase necessitates an update of the ancestor chart.

Best,
Kristian
